Cargo.toml

```diff
@@ -35,9 +35,13 @@ members = [
   "libs/user-facing-errors",
 ]
 
+[profile.dev]
+split-debuginfo = "unpacked"
```

Comment:
This is a bit obscure if you don't have context. Can we make it a separate commit with a message that links to the blog post for explanation? (or maybe a comment)

Reply:
Yep, will do